snowpack/src/scan-imports.ts

```diff
@@ -208,55 +240,13 @@ export async function scanImports(cwd: string, config: SnowpackConfig): Promise<
         return null;
       }
 
-      switch (baseExt) {
-        // Probably a license, a README, etc
-        case '': {
-          return null;
-        }
-        // Our import scanner can handle normal JS & even TypeScript without a problem.
-        case '.js':
-        case '.jsx':
-        case '.mjs':
-        case '.ts':
-        case '.css':
-        case '.tsx': {
-          return {
-            baseExt,
-            expandedExt,
-            locOnDisk: filePath,
-            contents: await fs.promises.readFile(filePath, 'utf-8'),
-          };
-        }
-        case '.html':
-        case '.vue':
-        case '.svelte': {
```

Comment:
This entire switch block didn’t go away; it just moved into `parseFileForInstallTargets()`